jsk_perception/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ find_package(catkin REQUIRED COMPONENTS
   angles
   cmake_modules
   cv_bridge
+  darknet
```

Comment:
I tried to compile with darknet from deb, but it seems to be missing compile settings from my another PR ( https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/83 ) and I could not compile at all.
I'm now continuing improving this PR with `jsk_3rdparty` version of `darknet` package.